Modix/ModixBot.cs

```diff
@@ -184,7 +184,11 @@ public ModixBot(ModixConfig config, ILogger logger)
             _map.AddSingleton<InviteLinkHandler>();
             _map.AddScoped<IBehaviourConfigurationRepository, BehaviourConfigurationRepository>();
             _map.AddScoped<IBehaviourConfigurationService, BehaviourConfigurationService>();
-            _map.AddSingleton<IBehaviourConfiguration, Services.BehaviourConfiguration.BehaviourConfiguration>();
+            _map.AddSingleton<IBehaviourConfiguration, BehaviourConfiguration>();
+
+            _map.AddSingleton<INoteCreatorService, NoteCreatorService>();
```

Comment:
Wrap these up in an extension method. See `ModerationSetup` and `CoreSetup`, for example.